image_classification/inception_v4.py

```diff
@@ -0,0 +1,526 @@
+    b3_conv2 = img_conv(
+        name='inceptA{0}_branch3_conv2'.format(depth),
+        input=b3_conv1,
+        num_channels=96,
+        num_filters=96,
+        filter_size=3,
+        stride=1,
+        padding=1)
+    return paddle.layer.concat(input=[b0_conv0, b1_conv0, b2_conv1, b3_conv2])
+
+
+def Inception_B(input, depth):
+    b0_pool0 = paddle.layer.img_pool(
+        name='inceptB{0}_branch0_pool0'.format(depth),
+        input=input,
+        num_channels=1024,
+        pool_size=3,
+        stride=1,
+        padding=1,
+        pool_type=paddle.pooling.Avg())
+    b0_conv0 = img_conv(
+        name='inceptB{0}_branch0_conv0'.format(depth),
+        input=b0_pool0,
+        num_channels=1024,
+        num_filters=128,
+        filter_size=1,
+        stride=1,
+        padding=0)
+    b1_conv0 = img_conv(
+        name='inceptB{0}_branch1_conv0'.format(depth),
+        input=input,
+        num_channels=1024,
+        num_filters=384,
+        filter_size=1,
+        stride=1,
+        padding=0)
+    b2_conv0 = img_conv(
+        name='inceptB{0}_branch2_conv0'.format(depth),
+        input=input,
+        num_channels=1024,
+        num_filters=192,
+        filter_size=1,
+        stride=1,
+        padding=0)
+    b2_conv1 = img_conv(
+        name='inceptB{0}_branch2_conv1'.format(depth),
+        input=b2_conv0,
+        num_channels=192,
+        num_filters=224,
+        filter_size=(1, 7),
+        stride=1,
+        padding=(0, 3))
+    b2_conv2 = img_conv(
+        name='inceptB{0}_branch2_conv2'.format(depth),
+        input=b2_conv1,
+        num_channels=224,
+        num_filters=256,
+        filter_size=(7, 1),
+        stride=1,
+        padding=(3, 0))
+    b3_conv0 = img_conv(
+        name='inceptB{0}_branch3_conv0'.format(depth),
+        input=input,
+        num_channels=1024,
+        num_filters=192,
+        filter_size=1,
+        stride=1,
+        padding=0)
+    b3_conv1 = img_conv(
+        name='inceptB{0}_branch3_conv1'.format(depth),
+        input=b3_conv0,
+        num_channels=192,
+        num_filters=192,
+        filter_size=(1, 7),
+        stride=1,
+        padding=(0, 3))
+    b3_conv2 = img_conv(
+        name='inceptB{0}_branch3_conv2'.format(depth),
+        input=b3_conv1,
+        num_channels=192,
+        num_filters=224,
+        filter_size=(7, 1),
+        stride=1,
+        padding=(3, 0))
+    b3_conv3 = img_conv(
+        name='inceptB{0}_branch3_conv3'.format(depth),
+        input=b3_conv2,
+        num_channels=224,
+        num_filters=224,
+        filter_size=(1, 7),
+        stride=1,
+        padding=(0, 3))
+    b3_conv4 = img_conv(
+        name='inceptB{0}_branch3_conv4'.format(depth),
+        input=b3_conv3,
+        num_channels=224,
+        num_filters=256,
+        filter_size=(7, 1),
+        stride=1,
+        padding=(3, 0))
+    return paddle.layer.concat(input=[b0_conv0, b1_conv0, b2_conv2, b3_conv4])
+
+
+def Inception_C(input, depth):
```

Comment:
Similar to `Inception_A`,  the network structure here has some diffs in with the TF implementation of `Inception_C`, like `branch0` excludes `pooling` and `branch3` includes `pooling`.

Reply:
Same as above